source/documentation.html.md

```diff
@@ -13,7 +13,7 @@ search: true
 
 # A Warm Welcome
 
-Hello there, if you are looking for some help on using the Sleepy Discord Library, then you came to the right place (I hope it is). If you have any questions you can ask me ( Sleepy Flower Girl ) on discord; right now there is no official server for this library, but you can alway find me on the Discord api server. If you like to help, you can always make a pull request for the docs or the library itself on github. Thanks!
+Hello there, if you are looking for some help on using the Sleepy Discord Library, then you came to the right place (I hope it is). If you have any questions you can ask me ( Sleepy Flower Girl ) on Discord; right now there is no official server for this library, but you can alway find me on the Discord server. If you like to help, you can always make a pull request for the docs or the library itself on github. Thanks!
```

Comment:
> but you can alway find me on the Discord server

can you change that to

> but you can always find me on the Discord API server

maybe add in the invite link as well, but I think people can look that up themselves